spec/controller/accounts_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe AccountsController, type: :controller do
+  describe "GET #new" do
+    before{get :new}
+
+    context "Return to success response" do
```

Comment:
context dùng ntn e nhỉ

Reply:
context dùng trong trưởng hợp có if ạ
ví dụ như if loged in => context "loged in"